llvm/test/Transforms/SLPVectorizer/AMDGPU/vectorize-i8-as-i32.ll

```diff
@@ -0,0 +1,1261 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
The name "vectorize-i8-as-i32.ll" is misleading I think -- can you rename to "vectorize-i8.ll"